packages/coding-standards/composer.json

```diff
@@ -23,7 +23,7 @@
         "squizlabs/php_codesniffer": "^3.5.0",
         "symplify/package-builder": "^7.3.18",
         "symplify/easy-coding-standard": "^7.3.18",
-        "symfony/finder": "^4.4|5.0"
```

Comment:
Hi, I found out during testing the package cannot be properly installed on symfony 5+ app due to conflict section in composer.json.
```
  "conflict": {
        "symfony/dependency-injection": ">=4.4.19"
    },
```

This should probably be closed interval in this package at least. 
(Try creating website skeleton of Symfony 5 project and require shopsys/coding-standards there)

Reply:
Actually it cannot be closed interval becasuse of `symplify/easy-coding-standard`. In `symfony/dependency-injection` was changed some private properties to constans (if i remember it right) which lead to BC break in ECS.

I see this PR is probably named wrong but we cannot do much about it now.

Reply:
Good point, thanks! Fixed in 24dac18835aec8d3142f7c0d4b96313d537524d5